### PR TITLE
Fixes #890 - make role macro private

### LINKED ISSUE #890: Make role-macro private
`/the-bureau/bureau-structure/role-macro.html` should return a 404, but is accessing a macro instead. 

Rename `role-macro.html` `_macro-role.html` and move it to the root `/the-bureau/` folder, following the structure of `/careers/`. Be sure to update the imports also.

### PR DESCRIPTION
**Pull request by anselmbradford**
Fixes #890 - make role macro private.

## Changes

- Made `/the-bureau/bureau-structure/role-macro.html` private.
- Updates method to be `render` instead of `display_role`.
- Added code comment for role macro.

## Testing

- `/the-bureau/bureau-structure/` should work the same.

## Review

- @jimmynotjim 
- @sebworks 
- @KimberlyMunoz

### CODE REVIEW

**Comment by sebworks** [comment]
Did you mean to include `src/the-bureau/_macro-role.html`?

**Comment by anselmbradford** [comment]
> Did you mean to include src/the-bureau/_macro-role.html?

Yes, but argh, meant to delete `role-macro.html` Fixed in https://github.com/cfpb/cfgov-refresh/commit/a915b6aef6126332dccfc7579b641554ebdd4345

File: src/the-bureau/_macro-role.html [hunk 1]
```diff
@@ -0,0 +1,33 @@
+{# ==========================================================================
+
+   role.render()
+
+   ==========================================================================
+
+   Description:
+
+   Renders a time element with a standard datetime format when given:
```
Comment (sebworks) [inline src/the-bureau/_macro-role.html]: Is this description correct?

File: src/the-bureau/_macro-role.html [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+   datetime:             A date timestamp to format.
```
Comment (sebworks) [inline src/the-bureau/_macro-role.html]: same here?

File: src/the-bureau/_macro-role.html (hunk 1 above)
Comment (anselmbradford) [inline src/the-bureau/_macro-role.html]: Argh! Nope. I accidentally had the right code comment in the file I needed to delete and not vice versa. Fixed in https://github.com/cfpb/cfgov-refresh/commit/877d7e3e0a9e78c0c4cee90b6e20c90949b1dc1e

File: src/the-bureau/_macro-role.html (hunk 2 above)
Comment (anselmbradford) [inline src/the-bureau/_macro-role.html]: Fixed in https://github.com/cfpb/cfgov-refresh/commit/877d7e3e0a9e78c0c4cee90b6e20c90949b1dc1e

**Comment by sebworks** [comment]
:+1:

**Pull request merged**